content/documentation/pages/6-recipes/2-file-ingest/1-sftp-to-jdbc.md

```diff
@@ -0,0 +1,936 @@
+The request sets `localFilePath` as a command line argument for the task. When running on a cloud platform, we need to mount a shared directory available to the SFTP source container and the task container.
+For this example, we will set up an NFS mounted directory.
+Configuring the environment and containers for NFS is platform specific and is described here for Cloud Foundry v2.3+ and minikube.
+
+## Prerequisites
+
+### Data Flow Installation
+
+Make sure you have installed Spring Cloud Data Flow to the platform of your choice:
+
+- [Local](%currentPath%/installation/local/)
+- [Cloud Foundry](%currentPath%/installation/cloudfoundry)
+- [Kubernetes](%currentPath%/installation/kubernetes/)
+
+<!--NOTE-->
+
+**NOTE**: For kubernetes, the sample task application is configured to use `mysql`. The Data Flow server must also be configured for mysql.
+
+**TODO**: Support for Postgres
+
+<!--END_NOTE-->
+
+### Using Data Flow
+
+This example assumes that you know how to use Spring Cloud Data Flow to register and deploy applications using the Spring Cloud Data Flow dashboard or the Spring Cloud Data Flow shell. If you need further instructions on using Data Flow please refer to [Stream Processing using Spring Cloud Data Flow](%currentPath%/stream-developer-guides/streams/data-flow-stream) and [Register and launch a batch application using Spring Cloud Data Flow](%currentPath%/batch-developer-guides/batch/data-flow-spring-batch/).
+
+### SFTP server
+
+This example requires access to an SFTP server. For running on a `local` machine and `minikube`, we will use the host machine as the SFTP server. For `Cloud Foundry`, and `Kubernetes` in general, an external SFTP server is required.
+On the SFTP server, create a `/remote-files` directory. This is where we will drop files to trigger the pipeline.
+
+### NFS configuration
+
+<!--TIP-->
+
+NFS is not required when running locally.
+
+<!--END_TIP-->
+
+#### Cloud Foundry NFS configuration
+
+This feature is provided in Pivotal Cloud Foundry by [NFS Volume Services](https://docs.pivotal.io/pivotalcf/2-5/devguide/services/using-vol-services.html)
+
+To run this example, we will need:
+
+- a Cloud Foundry instance v2.3+ with NFS Volume Services [enabled](https://docs.pivotal.io/pivotalcf/2-5/opsguide/enable-vol-services.html)
+- An NFS server accessible from the Cloud Foundry instance
+- An `nfs` service instance properly configured
+
+<!--NOTE-->
+
+**NOTE:** For simplicity, this example assumes the `nfs` service is created with common configuration as follows with a common mount point `/var/scdf` for all bound apps. It is also possible to set these parameters when binding the nfs service to an application using [deployment propterties](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#configure-service-binding-parameters):
+
+```bash
+$cf create-service nfs Existing nfs -c '{"share":<nfs-host>/staging","uid":<uid>,"gid":<gid>, "mount":"/var/scdf"}'
```

Comment:
It'd be good to remove `$` from all the bash commands. O'wise, when copying from the Microsite shortcut, everything here gets copied in entirety.